packages/vuetify/src/util/dateTimeUtils.ts

```diff
@@ -0,0 +1,56 @@
+const DAY_OF_YEAR_PER_FIRST_OF_MONTH = [0, 31, 59, 90, 120, 151, 181, 212, 243, 273, 304, 334]
```

Comment:
This needs a style / readability pass.

Reply:
Restored array on its original position in method body. But I couldn't find anything in the coding guidelines which I could use for this. Maybe you can point me out where I can find it.